### ISSUE TITLE
Misaligment of minimum PHP Version and Composer 2 platform_check

### ISSUE THREAD

**Issue by anibalsanchez** [akeeba/loginguard#105]
Please read the README.md in the repository's root and the support resources before submitting an issue.

### Steps to reproduce the issue

The pkg_loginguard-4.0.0.zip has compatibility with minimum PHP Version 7.2. However, it also contains a Composer Platform check of 7.3 (com_loginguard/vendor/composer/platform_check.php => laminas/laminas-diactoros => ^7.3 || ~8.0.0).

If the package is installed on a PHP 7.2, it shows a blank page with this error:

```
Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 7.3.0".
```

### Expected result

The usual login

### Actual result

Blank page

```
Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 7.3.0".
```

### Troubleshooting already performed

It looks like Composer 2 is checking all libraries and determining that PHP version ">= 7.3.0". I think that it is coming from the laminas/laminas-diactoros dependency.

### System information

- loginguard-4.0.0
- PHP 7.2

#### Mandatory information

Issues without this information can not and _will not_ be replied to.

* **Joomla! version**: 3.9.26
* **PHP version**: 7.2.24
* **Am I using HTTPS**: Yes
* **Server Operating System**: Linux

**Comment by nikosdion** [comment]
The only fix to that would be dropping PHP 7.2 completely. This is pretty much the case; I do not have a working PHP 7.2 environment anymore.

FWIW, I am already working on LoginGuard 5 which uses core MVC. I'll do the version bump there.

**Comment by anibalsanchez** [comment]
Great.
